src/styles.scss

```diff
@@ -2,42 +2,33 @@ html {
     font-size: 14px;
 }
 
-@mixin font-face($Name, $ext) {
+@mixin font-face($Name, $extArray) {
+
     @font-face {
+
         font-family: $Name;
-        font-weight: normal;
-        src: url("fonts/" + $Name + "." + $ext) format($ext);
+        @each $ext in $extArray  {
+             @if $ext == "ttf" {
```

Comment:
No need to create universal solution when you use only woff and woff2. Just leave last else here